Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1646,7 +1647,8 @@ void CGContextFillRects(CGContextRef context, const CGRect* rects, size_t count)
 */
 void CGContextDrawPath(CGContextRef context, CGPathDrawingMode mode) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
+
+    __CGContextDrawGeometry(context, _getAndClosePathGeometry(context->Path()), mode);
```

Comment:
>__CGContextDrawGeometry [](start = 4, length = 23)

We may need to invert the user transform on this draw, because otherwise we could apply a double-transformed path.